keps/sig-node/3983-drop-in-configuration/README.md

```diff
@@ -220,11 +220,12 @@ All behavior change is encapsulated within the Kubelet, so there is no version s
 
 ###### How can this feature be enabled / disabled in a live cluster?
 
-- [x] Feature gate
-  - Feature gate name: KubeletDropInConfig
+- [] Feature gate
   - Components depending on the feature gate: Kubelet
 
-Aside from the featuregate, admins will also have to explicitly enable the kubelet flag to enable this.
+Aside from setting up the environment variable `KUBELET_CONFIG_DROPIN_DIR_ALPHA`, admins will also need to explicitly set the --config-dir flag in the kubelet's CLI to enable this.
+
+The decision to use an environment variable (KUBELET_CONFIG_DROPIN_DIR_ALPHA) over a feature gate was made to avoid potential conflicts in configuration settings. With the current configuration flow, feature gates could lead to unexpected behavior when CLI settings conflict with the kubelet.conf.d directory. The potential issue arises when the CLI initially sets the feature gate to "off," but the kubelet configuration specifies it as "on." In this scenario, the kubelet would start with the feature gate "off," switch it to "on" during configuration rendering, and then have conflicting settings when reading the kubelet.conf.d directory, leading to unexpected behavior. By using an environment variable during the alpha phase, we provided a simpler and more predictable way to control the drop-in configuration directory for testing. In the beta phase, we are removing this environment variable to streamline configuration management and enhance the user experience.
```

Comment:
Since you are removing the environment variable now, specifying the `--config-dir` is the only flag? The default is still  "" and defaults to disabled?

Reply:
Yes, that's the correct interpretation. Should this information be explicitly stated in this paragraph?